### ISSUE TITLE
Add date of introduction for Columba livia

### ISSUE THREAD

**Issue by damianooldoni** [trias-project/alien-birds-checklist#11]
While working on occurrence indicators I noticed that the datacube has a taxon which refers to the pigeon, `Columba livia`.

Moving backwards in the processing chain I found this taxon in the unified checklist and there I found that it has been published in the Alien birds checklist:
https://www.gbif.org/species/159498341

I am not an expert and I didn't know it is an alien species in Belgium. @timadriaens ? Any comment about?

By the way, it seems it is considered alien even in GRIIS of Denmark and UK:
https://www.gbif.org/species/148742092
https://www.gbif.org/species/148704311

**Comment by timadriaens** [comment]
That is the scientific name for feral pigeon (C. livia f. domestica), supposed to be descended from the wild rock pigeon C. livia

**Comment by timadriaens** [comment]
We do it just the way ebird does, see here https://ebird.org/news/rock-pigeon/

**Comment by damianooldoni** [comment]
Putting Columba Livia in alien birds checklist WITHOUT date of introduction means that this species is included in the emerging status analysis. 
And as you can see in [GAM output](https://github.com/trias-project/indicators/blob/occurrence-indicators/data/output/GAM_outputs/GAM_correct_baseline_2495414_Columba%20livia.png) it appears to be definitely emerging. If this it's ok to you, then great! Just close the issue.
I have never thought the feral pigeon was a "recent" alien species. Just that... The ignorance of a non-expert. :nerd_face:

**Comment by timadriaens** [comment]
Most probably an artefact of people noting it more. Anyway, with revisions and additions to the bird checklist this should be solved as it is long established (mind that we still need to verify whether the 1950 cut off is a good one, as some species can have much longer lag times). The prioritisation tool is a decision support tool and expert input will always be necessary when using it.

**Comment by qgroom** [comment]
Yes, I was thinking it might be the app effect. We record everywhere any time now, whereas before we went out to places specifically to record. My most recorded animals on iNaturalist are 1: _Columba livia_, 2: _Harmonia axyridis_ & 3: _Talpa europaea_. There are certainly some biases going on there.

**Comment by damianooldoni** [comment]
Ok. I understand by your comments, @timadriaens, that Columba livia is really an alien species. Thanks, I didn't know. I strongly suggest to add a date of introduction (see change of title of this issue). The more expert knowledge is present in the data (as part of the checklists), the best we can assess emerging status based on data and the less expert knowledge is needed afterwards. 
If an expert, asked to review the list of emerging species, says: "This species is long time present in Belgium, we cannot do anything to eradicate it and we are even not interested to do it", this means we could add this expert knowledge in the data.

@qgroom : my top three: 1. _Taraxacum officinale_,2.  _Columba livia domestica_, 3. _Senecio vulgaris._  At least two out of three are native :smile: 
Yes, there are bias, the first bias is just the way we see the world: we record observations of plants and animals which can be seen by eyes. No bacteria, microscopical fungi etc. :microscope:

**Comment by timadriaens** [comment]
as with all the checklists, adding more data would be good but it requires searching for books or articles, consulting experts etc.

**Comment by damianooldoni** [comment]
I know @timadriaens. I am a data lover, you know. It's my role to be the [talking cricket](https://en.wikipedia.org/wiki/Talking_Cricket) when it comes to strange stuff like this.

The funny stuff is that it takes (a lot of) time to introduce a date of introduction in a checklist, but it takes less than a second to remove this species from the list of emerging species worth a risk assessment :smile:
